.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,34 @@
+## Contributing to Resource Hub
+
+### Reporting a bug:
+
+Check if the bug was already reported in the [Github Issues](https://github.com/resource-hub-dev/rhub-api/issues).
+
+If it is not there, feel free to open a [new issue](https://github.com/resource-hub-dev/rhub-api/issues/new/choose). Make sure to follow
+the template and include all relevant information.
+
+If you want to:
+* suggest some new feature/functionality to the team
+* discuss some new feature/functionality you want to implement before starting the actual development
+* ask any question related to features/functionalities
+
+then please follow the appropriate template while creating the issue, and provide as much information as possible.
+
+### Sending a pull request with a patch/feature/fix:
+
+To send a pull request:
+* Make sure to have opened a bug report / feature request / question before, so it can be related and validated
+* Fork the project
+* Commit your changes. Please provide concise commits, with a reasonable and descriptive commit message.
+* Follow the pull request template, making sure to provida as much information as needed.
+
+Pull requests will be accepted when, and only when:
+* All of the test suites are passing
+* The changeset is approved by one or more team members
+
+The preferred method will be `rebase and merge`. If the commit message(s) need to be adjusted or clarified, we might `squash`
```

Comment:
This just seems to come out of nowhere.

Reply:
Reworded it so it's simpler and makes more sense.